tfctl/uninstall.go

```diff
@@ -0,0 +1,101 @@
+package tfctl
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	"github.com/fluxcd/pkg/ssa"
+	"github.com/theckman/yacspin"
+	appsv1 "k8s.io/api/apps/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// Uninstall removes the tf-controller resources from the cluster.
+func (c *CLI) Uninstall(out io.Writer) (retErr error) {
```

Comment:
I don't think you need to name the return here (you don't actually use it in the code), and `err` is the expected name of errors being returned.

Reply:
I wanted to check for errors in a defer function which cleans up the spinner so added the named return here.